exe/task.cc

```diff
@@ -198,10 +364,19 @@ int MAIN_IMPL(int argc, char** argv)
     auto exitcode = 255;
     lth_jc::JsonContainer stdout_result;
     string stderr_str;
-
+ 
     try {
         task_runner runner;
-        auto task = runner.find_task(taskname);
+        task task;
+        if (!args.includes("files")) {
+            task = runner.find_task(taskname);
+        } else {
+            // TODO: Do I need to check if files has at least one element?
```

Comment:
We'll need to check that somewhere, so we don't crash if it doesn't. Probably a PXP error again if it doesn't.